README.tmpl.md

```diff
@@ -108,13 +108,11 @@ docker run --rm -v $(pwd):/app -w /app golangci/golangci-lint:{{.LatestVersion}}
 
 ### Go
 
-Please, do not install `golangci-lint` by `go get`:
+Go source installations are supported for the two most recent Go releases.
 
-1. [`go.mod`](https://github.com/golangci/golangci-lint/blob/master/go.mod) replacement directive doesn't apply. It means you will be using patched version of `golangci-lint`.
-2. it's much slower than binary installation
-3. its stability depends on your Go version (e.g. on [this compiler Go <= 1.12 bug](https://github.com/golang/go/issues/29612)).
-4. it's not guaranteed to work: e.g. we've encountered a lot of issues with Go modules hashes.
-5. it allows installation from `master` branch which can't be considered stable.
+```bash
+GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
Please use `@{{.LatestVersion}}`, this command will <kbd>Ctrl</kbd> + <kbd>C</kbd>  <kbd>Ctrl</kbd> + <kbd>V</kbd> and we should use version pinning.

Reply:
It's not clear to me how pinning a dynamically defined string is better than using the `go get` behaviour which would do the same thing. But OK.

Reply:
> It's not clear to me how pinning a dynamically defined string is better than using the go get behaviour which would do the same thing. But OK.

Sorry for being unclear. I'm talking about running this command in CI, like that:
```yaml
lint:
  run: GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@1.24.0
```

You are right, there is no difference outside this case.